db/migrate/20150326121012_add_missing_migrations.rb

```diff
@@ -0,0 +1,15 @@
+class AddMissingMigrations < ActiveRecord::Migration
+  def change
+    create_table :flowdock_flows do |t|
```

Comment:
Yes, good point, I found the DB migrations in the plugin directories. I'll have to figure out why it didn't pick those up when regenerating the schema.